edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesRuntimeInfoProviderTest.cs

```diff
@@ -142,7 +167,7 @@ public async Task ReturnsLastKnowModuleState()
             var runtimeInfo = new KubernetesRuntimeInfoProvider(Namespace, client.Object, moduleManager.Object);
             foreach ((string podName, var pod) in BuildPodList())
```

Comment:
One benefit of a good IDE is to show unused code and variables. For example, `podName` is not used enywhere her. so code can be improved:
```suggestion
             foreach (V1Pod pod in BuildPodList().Values)
```